packages/inferno-from-react-generator/src/expressions/inferno-hooks-variable-statement-wrapper.ts

```diff
@@ -21,13 +23,22 @@ export class InfernoHooksVariableStatementWrapper extends VariableStatement {
     super([], declarationList);
     this.hasExport = modifiers.indexOf(SyntaxKind.ExportKeyword) !== -1;
     this.componentInfo = componentInfo;
-    componentDeclaration.name = new Identifier(`React${componentDeclaration.name.toString()}`);
-    componentDeclaration.initializer = (componentDeclaration.initializer as Call)
-      .argumentsArray[0];
+    if (componentDeclaration.initializer instanceof Call) {
+      const call = componentDeclaration.initializer;
```

Comment:
```suggestion
      const call = componentDeclaration.initializer as Call;
```
Btw is this cast really needed? Doesn't type narrowing work here?